cmd/eksctl-anywhere/cmd/options.go

```diff
@@ -155,10 +161,30 @@ func newClusterSpec(options clusterOptions) (*cluster.Spec, error) {
 	return clusterSpec, nil
 }
 
-func markFlagHidden(flagSet *pflag.FlagSet, flagName string) {
-	if err := flagSet.MarkHidden(flagName); err != nil {
-		logger.V(5).Info("Warning: Failed to mark flag as hidden: " + flagName)
-	}
+func newEKSARelease(bundles *releasev1.Bundles, options clusterOptions) (*releasev1.EKSARelease, error) {
+	opts := fileSpecBuilderOptsFromClusterOptions(options)
+
+	cliVersion := version.Get()
+	b := cluster.NewFileSpecBuilder(
+		files.NewReader(files.WithEKSAUserAgent("cli", cliVersion.GitVersion)),
+		cliVersion,
+		opts...,
+	)
+
+	return cluster.BuildEKSARelease(b, bundles)
+}
+
+func newBundles(options clusterOptions) (*releasev1.Bundles, error) {
+	opts := fileSpecBuilderOptsFromClusterOptions(options)
+
+	cliVersion := version.Get()
+	b := cluster.NewFileSpecBuilder(
+		files.NewReader(files.WithEKSAUserAgent("cli", cliVersion.GitVersion)),
+		cliVersion,
+		opts...,
+	)
+
+	return cluster.GetBundlesManifest(b)
 }
```

Comment:
I would advise to build the spec builder with one method and reuse it from the cmd to get the two objects you want